<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>680</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label{
	
	background-color: rgb(93, 255, 125);
	border: 4px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_2{
	
	background-color: rgb(93, 255, 125);
	border: 4px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_3{
	
	background-color: rgb(93, 255, 125);
	border: 4px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_4{
	
	background-color: rgb(93, 255, 125);
	border: 4px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_5{
	
	background-color: rgb(93, 255, 125);
	border: 4px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_6{
	
	background-color: rgb(255, 0, 0);
	border: 2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_7{
	
	background-color: rgb(255, 0, 0);
	border: 2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_8{
	
	background-color: rgb(255, 0, 0);
	border: 2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_9{
	
	background-color: rgb(255, 0, 0);
	border: 2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_10{
	
	background-color: rgb(255, 0, 0);
	border: 2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
